site/documentation/content/deployment/create-kubernetes-cluster.md

```diff
@@ -19,214 +21,22 @@ Please refer to Kubernetes' [Minikube setup guide](https://kubernetes.io/docs/se
 
 The recommended settings for a Minikube VM used for running Hono's example setup are as follows:
 
-* **cpus**: 2
-* **memory**: 8192 (MB)
+- **cpus**: 2
+- **memory**: 8192 (MB)
 
 The command to start the VM will look something like this:
+
 ```sh
 minikube start --cpus 2 --memory 8192
 ```
 
-After the Minikube VM has started successfully, the `minikube tunnel` command should be run in order to support Hono's services being deployed using the *LoadBalancer* type. Please refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#access-to-loadbalancer-services-using-minikube-tunnel) for details.
+After the Minikube VM has started successfully, the `minikube tunnel` command should be run in order to support Hono's services being deployed using the _LoadBalancer_ type. Please refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#access-to-loadbalancer-services-using-minikube-tunnel) for details.
```

Comment:
I do not see the point of this change. Can you either explain or revert?

Reply:
Yeah, its my editor. Sry about that. I have to disable _pretty print_ :)